medusa/name_parser/parser.py

```diff
@@ -163,10 +163,21 @@ def _parse_string(self, name):
                 new_episode_numbers.append(episode)
                 new_season_numbers.append(season)
 
-        elif result.show.is_anime and result.is_anime:
+        # result.is_anime, means that guessit detected the episode like an absolute number.
+        elif result.show.is_anime and (result.is_anime or (result.season_number is None and result.episode_numbers)):
             log.debug('Scene numbering enabled series {name} is anime',
                       {'name': result.show.name})
-            scene_season = scene_exceptions.get_scene_exception_by_name(result.series_name)[1]
+
+            if result.season_number is None and not result.ab_episode_numbers and result.episode_numbers:
```

Comment:
Do you think this is necessary?

Reply:
I'm not sure. I'm giving this PR #3151 priority. If we've fixed that, i'm going back to this.